airbyte-integrations/connectors/source-ashby/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "0.1.3"
+version = "0.2.0"
```

Comment:
Please run `poetry lock` in the connector folder to regenerate the dependency file.